### PR TITLE
Fix sorting bug and update eth client

### PR DESCRIPTION
**Pull request by yjcbert**
- [x] update describe of query consensus state
- [x] update header errors
- [x] todo

### CODE REVIEW

File: modules/tibc/core/24-host/keys.go [hunk 1]
```diff
@@ -94,8 +96,18 @@ func ConsensusStatePath(height exported.Height) string {
 
 // ConsensusStateKey returns the store key for a the consensus state of a particular
 // client stored in a client prefixed store.
-func ConsensusStateKey(height exported.Height) []byte {
-	return []byte(ConsensusStatePath(height))
+//func ConsensusStateKey(height exported.Height) []byte {
+//	return []byte(ConsensusStatePath(height))
+//}
```
Comment (chengwenxi) [inline modules/tibc/core/24-host/keys.go:101]: delete

File: modules/tibc/light-clients/09-eth/types/client_state.go [hunk 2]
```diff
@@ -6,6 +6,8 @@ import (
 	"encoding/json"
 	"fmt"
 
+	clienttypes "github.com/bianjieai/tibc-go/modules/tibc/core/02-client/types"
+
```
Comment (chengwenxi) [inline modules/tibc/light-clients/09-eth/types/client_state.go:10]: move to */tibc-go/*

**Review by chengwenxi — COMMENTED** [review]
Seems should update proto file

**Review by chengwenxi — APPROVED** [review]

File: simapp/app.go [hunk 3]
```diff
@@ -179,7 +180,7 @@ type SimApp struct {
 	CrisisKeeper      crisiskeeper.Keeper
 	UpgradeKeeper     upgradekeeper.Keeper
 	ParamsKeeper      paramskeeper.Keeper
-	TIBCKeeper        *tibckeeper.Keeper // TIBC Keeper must be a pointer in the app, so we can SetRouter on it correctly
+	TIBCKeeper        *tibckeeper.Keeper // IBC Keeper must be a pointer in the app, so we can SetRouter on it correctly
```
Comment (SegueII) [inline simapp/app.go:183]: IBC => TIBC

File: simapp/app.go [hunk 4]
```diff
@@ -259,8 +260,9 @@ func NewSimApp(
 	// set the BaseApp's parameter store
 	bApp.SetParamStore(app.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramskeeper.ConsensusParamsKeyTable()))
 
-	// add capability keeper and ScopeToModule for tibc module
+	// add capability keeper and ScopeToModule for ibc module
```
Comment (SegueII) [inline simapp/app.go:263]: tibc

File: simapp/app.go [hunk 5]
```diff
@@ -301,7 +303,7 @@ func NewSimApp(
 		stakingtypes.NewMultiStakingHooks(app.DistrKeeper.Hooks(), app.SlashingKeeper.Hooks()),
 	)
 
-	// Create TIBC Keeper
+	// Create IBC Keeper
```
Comment (SegueII) [inline simapp/app.go:306]: TIBC

File: simapp/app.go [hunk 6]
```diff
@@ -328,8 +330,8 @@ func NewSimApp(
 	)
 	nfttransferModule := tibcnfttransfer.NewAppModule(app.NftTransferKeeper)
 
-	// NOTE: the TIBC mock keeper and application module is used only for testing core TIBC. Do
-	// note replicate if you do not need to test core TIBC or light clients.
+	// NOTE: the IBC mock keeper and application module is used only for testing core IBC. Do
+	// note replicate if you do not need to test core IBC or light clients.
```
Comment (SegueII) [inline simapp/app.go:334]: TIBC

File: modules/tibc/core/02-client/keeper/grpc_query_test.go [hunk 7]
```diff
@@ -282,6 +282,7 @@ func (suite *KeeperTestSuite) TestQueryConsensusStates() {
 				},
 			}
 		},
+
```
Comment (SegueII) [inline modules/tibc/core/02-client/keeper/grpc_query_test.go:285]: Remove blank lines

**Review by mindcarver — APPROVED** [review]

**Pull request merged**